SDL_ttf.h

```diff
@@ -2311,6 +2311,21 @@ extern DECLSPEC int SDLCALL TTF_SetFontDirection(TTF_Font *font, TTF_Direction d
  */
 extern DECLSPEC int SDLCALL TTF_SetFontScriptName(TTF_Font *font, const char *script);
 
+/**
+ * Retrieves the script to which unicode character belongs.
+ *
+ * The supplied script value must be exactly four characters.
```

Comment:
bad documentation: this new procedure _retrieves_ the script value. Should be something like `The supplied script pointer should be able to hold four characters.` And if we want to null-terminate it, adjust it accordingly, e.g. like `The supplied script pointer should be able to hold four characters and the null-terminator.`

Reply:
Resolved, added on the description